File: Beginner/1072 - Interval 2/1072.c
#include <stdio.h>
 
int main() {
 
    double aux;
    int casos, i, in=0, out=0;
    
    scanf("%i", &casos);
    
    for(i=0; i<casos; i++){
        scanf("%lf", &aux);
        
        if(aux>=10 && aux<=20)
            in++;
        
        else
            out++;
    }
    
    printf("%i in\n", in);
    printf("%i out\n", out);
    
    return 0;
}

File: Beginner/1072 - Interval 2/1072.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int casos = int.Parse(Console.ReadLine());
        int inCount = 0;

        for (int i = 0; i < casos; i++)
        {
            int aux = int.Parse(Console.ReadLine());

            if (aux >= 10 && aux <= 20)
                inCount++;
        }

        Console.WriteLine($"{inCount} in");
        Console.WriteLine($"{casos - inCount} out");
    }
}